src/mediaplayer.cs

```diff
@@ -51,20 +51,27 @@ interface MPMediaItem : NSSecureCoding {
 
 #if XAMCORE_2_0
 	}
-
-	[NoMac]
-	[NoTV]
+#if MONOMAC || TVOS
+	[Static]
```

Comment:
Do macOS and tvOS have a different type hierarchy ? 
Otherwise this only makes code sharing more difficult across platforms